lib/util/flattenOperands.js

```diff
@@ -64,13 +60,13 @@ function flattenOperands(root) {
         // Rearrange multiplication of fractions involving variables in the
         // the numerator.
         // e.g. 2 x / 3 -> 2x / 3
-        else if (query.isNumber(node.args[0]) && query.isDiv(node.args[1])
-            && query.isIdentifier(node.args[1].args[0]) && node.implicit) {
+        // TODO (kevinb): only rewrites first two factors of multiplication
```

Comment:
It might be good to give an example of where this fails and describe what should happen in that case.